sdks/java/io/elasticsearch-tests/elasticsearch-tests-common/src/main/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOITCommon.java

```diff
@@ -34,6 +34,7 @@
  * <p>This is independent from the tests so that for read tests it can be run separately after data
  * store creation rather than every time (which can be more fragile.)
  */
+@SuppressWarnings("nullness")
```

Comment:
these test fixtures belong to main scope, and there is a backlog task eliminate "testRuntimeMigration" (#16307)